src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Startup.cs

```diff
@@ -124,6 +124,12 @@ public void ConfigureServices(IServiceCollection services)
                 options.Filters.Add(new AuthorizeFilter(policy));
```

Comment:
You would want to remove this. Basically we're replacing the filter with an auth policy, but you still want to keep MVC around.